_includes/v19.1/misc/external-urls.md

```diff
@@ -21,7 +21,7 @@ If your environment requires an HTTP or HTTPS proxy server for outgoing connecti
 
 <a name="considerations"></a>
 
-- <sup>1</sup> If the `AUTH` parameter is `implicit`, all GCS connections use Google's [default authentication strategy](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application). If the `AUTH` parameter is `default`, the `cloudstorage.gs.default.key` [cluster setting](cluster-settings.html) must be set to the contents of a [service account file](https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually) which will be used during authentication. If the `AUTH` parameter is not specified, the `cloudstorage.gs.default.key` setting will be used if it is non-empty, otherwise the `implicit` behavior is used.
+- <sup>1</sup>If the `AUTH` parameter is not specified, the `cloudstorage.gs.default.key` setting will be used if it is non-empty, otherwise the `implicit` behavior is used. If the `AUTH` parameter is `implicit`, all GCS connections use Google's [default authentication strategy](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application). If the `AUTH` parameter is `default`, the `cloudstorage.gs.default.key` [cluster setting](cluster-settings.html) must be set to the contents of a [service account file](https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually) which will be used during authentication. <span class="version-tag">New in v19.1:</span> If the `AUTH` paramenter is `specified`, GCS connections are authenticated on a per-statement basis, which allows the JSON key object to be sent in the `credentials` parameter.
```

Comment:
We should also mention that the key needs to be base64-encoded. Adding a sentence at the end like `The JSON key object should be base64-encoded (using the standard encoding in RFC 4648)` should be fine.

Also, I think the `CREDENTIALS` query param should be uppercase to match the table/the rest of the docs.

Reply:
Also, `parameter` is misspelled as `paramenter`.